src/main/java/org/victorrobotics/frc/dtlib/command/test/DTInstantTestCommand.java

```diff
@@ -0,0 +1,41 @@
+package org.victorrobotics.frc.dtlib.command.test;
+
+import org.victorrobotics.frc.dtlib.command.DTCommandBase;
+
+import edu.wpi.first.wpilibj2.command.Subsystem;
+
+public class DTInstantTestCommand extends DTCommandBase implements DTTestCommand {
+    private static final Runnable NULL_RUNNABLE = () -> {
+    };
+
+    private final Runnable action;
+
+    public DTInstantTestCommand() {
+        action = NULL_RUNNABLE;
```

Comment:
What's the use case for this constructor?

Reply:
Good point, with the `DTNullCommand` it’s basically useless